packages/block-editor/src/components/list-view/block.js

```diff
@@ -112,20 +111,13 @@ function ListViewBlock( {
 		level
 	);
 
-	let blockAriaLabel = __( 'Link' );
-	if ( blockInformation ) {
```

Comment:
I notice that the condition to check if `blockInformation` is truthy has been removed. From a little digging, I think the check was added back in https://github.com/WordPress/gutenberg/pull/38775 to deal with an issue with `useSelect` where it's possible that `blockInformation` will be `null` in some circumstances. So, I'm not too sure, but it might be necessary to keep the condition around. Just pinging @talldan who originally implemented the fix.

Reply:
If that is the case, please check the variable directly below this line. It was not included in this conditional and still used `blockInformation` to get the title for building the options `aria-label`.

Reply:
In the case of the below `settingsAriaLabel` it's doing a check for `blockInformation` within its ternary before attempting to access `blockInformation.title`, so it didn't need to be included in the `if` block. For `blockAriaLabel` both outcomes of the ternary are accessing properties of `blockInformation`, so if `blockInformation` were to be `null` then an error would be thrown.

Reply:
I'm not too sure if that check is still needed, though, but because #38775 introduced the check, it'd probaly  be good to verify before removing the check.

Reply:
Ah, yeah, I totally missed that sneaky conditional on that line. NVM.